src/Agent/NewRelic/Agent/Core/Utilization/VendorHttpApiRequestor.cs

```diff
@@ -66,5 +81,55 @@ public virtual string CallVendorApi(Uri uri, string method, string vendorName, I
                 return null;
             }
         }
+#else
+        // create a static HttpClient for use across all the vendor API calls
+        private static readonly HttpClient httpClient = new HttpClient
```

Comment:
If all the vendor checks are disabled, we won't need this, right? Maybe we could do a lazy init?

Reply:
Your right, I'll need to refactor it.